code/modules/awaymissions/gateway.dm

```diff
@@ -214,19 +222,24 @@ GLOBAL_DATUM(the_gateway, /obj/machinery/gateway/centerstation)
 	if(!active)
 		return
 	if(!stationgate || QDELETED(stationgate))
+		say("No Destination found.")
+		return
+	if(!stationgate.active)
+		say("Destination Gateway is not active")
 		return
 	if(isliving(AM))
```

Comment:
Still won't block them inside crates or other contents

Reply:
Might be simple enough I can make the time to address it in a reasonable timeframe. May be a week or two